plugin/src/helpers/edge.ts

```diff
@@ -86,10 +85,46 @@ const copyEdgeSourceFile = ({
 // Edge functions don't support lookahead expressions
 const stripLookahead = (regex: string) => regex.replace('^/(?!_next)', '^/')
 
+const writeEdgeFunction = async ({
+  edgeFunctionDefinition,
+  edgeFunctionRoot,
+  netlifyConfig,
+}: {
+  edgeFunctionDefinition: EdgeFunctionDefinition
+  edgeFunctionRoot: string
+  netlifyConfig: NetlifyConfig
+}): Promise<{
+  function: string
+  pattern: string
+}> => {
+  const name = sanitizeName(edgeFunctionDefinition.name)
+  const edgeFunctionDir = join(edgeFunctionRoot, name)
+
+  const bundle = await getMiddlewareBundle({
+    edgeFunctionDefinition,
+    netlifyConfig,
+  })
+
+  await ensureDir(edgeFunctionDir)
+  await fs.writeFile(join(edgeFunctionDir, 'bundle.js'), bundle)
+
+  await copyEdgeSourceFile({
+    edgeFunctionDir,
+    file: 'runtime.ts',
+    target: 'index.ts',
+  })
+
+  await copyEdgeSourceFile({ edgeFunctionDir, file: 'utils.ts' })
+  return {
+    function: name,
+    pattern: stripLookahead(edgeFunctionDefinition.regexp),
```

Comment:
Since we strip the lookahead, does this mean that there are some things we cannot support with this Edge function?